src/notify/app/validators/request_validator.py

```diff
@@ -25,15 +25,34 @@ def verify_headers(headers: dict, api_key: str) -> tuple[bool, str]:
 
     return True, ""
 
+
+def verify_get_statuses_headers(headers: dict, api_key: str) -> tuple[bool, str]:
```

Comment:
Echoing the point below about checks, I think this function and `verify_batch_headers` should be the same thing. ie the same header checks.
They should check for the consumer key header and the api key header, they don't need to check the HMAC signature header.

Reply:
ah ok, so both the batch and status request checks should look like below? or should the `verify_batch_headers` still check for the `authorization` header key?
```
def verify_consumer_get_request(headers: dict, api_key: str) -> tuple[bool, str]:
    lc_headers = header_keys_to_lower(headers)
    if lc_headers.get(API_KEY_HEADER_NAME) is None:
        return False, "Missing API key header"

    if lc_headers.get(API_KEY_HEADER_NAME) != api_key:
        return False, "Invalid API key"

    if lc_headers.get(CONSUMER_KEY_NAME) is None:
        return False, "Missing Consumer key header"

    return True, ""
  ```